src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/dbscripts/all_ri_children.sql

```diff
@@ -12,7 +12,18 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 
-SELECT *
+SELECT
+ IndexId AS "IndexId",
```

Comment:
Do we need "AS" here if the name does not change?

Reply:
Yes, the difference is that the name after the AS is in quotes and hence case sensitive while the first is treated cases insensitive. This way, we get the same behavior in HSQLDB and Teradata.